examples/injected-from-dependency/dub.json

```diff
@@ -0,0 +1,19 @@
+{
+	"description": "Some test code for Have_druntime version",
+	"name": "injected-from-dependency",
+	"targetType": "executable",
+
+	"dependencies": {
+		":toload": "*"
+	},
+
+	"subPackages": [
+		{
+			"name": "toload",
+			"buildOptions": ["betterC"],
+			"sourcePaths": ["toload"],
+			"importPaths": ["toload"],
+			"injectSourceFiles": ["ahook.d"]
+		}
+	]
```

Comment:
I think subPackages with injectSourceFiles sound a little unrealistic in real code. If I understand the purpose of this change correctly a user would depend on something on DUB which has an injectSourceFile set. Having this as an example could cause some confusion. (though for the test it's fine)

I would suggest making a sub-folder with an isolated package, (just move this content here into a dub.json in the toload folder) which is then used as a dependency with `{"path":"./toload"}` which would effectively be the same, but for documentation purposes is closer to what will actually happen on the field.